### PR TITLE
[Toolchain] Add yarn promote:production command

### PR DESCRIPTION
**Pull request by damassi**
Adds a `yarn promote:production` command which aliases `heroku pipelines:promote -r staging`.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
   "editor.rulers": [80],
   "editor.tabSize": 2,
+  "editor.formatOnSave": true,
+  "[json]": {
+    "editor.formatOnSave": false
```
Comment (damassi) [inline .vscode/settings.json]: Disable auto-formatting in our config files

File: README.md [hunk 2]
```diff
@@ -69,7 +84,8 @@ git remote add staging git@heroku.com:artsy-metaphysics-staging.git
 Then promote using the command:
 
 ```sh
-$ heroku pipelines:promote -r staging
+$ yarn promote:production
```
Comment (damassi) [inline README.md]: This is the only real change; the rest is Prettier formatting markdown.

Comment (orta) [reply README.md]: good good

**Comment by orta** [comment]
👍

**Pull request merged**